Gemfile

```diff
@@ -58,6 +58,11 @@ group :development do
   gem 'capistrano-passenger'
 end
 
+group :test do
```

Comment:
@lizgarciao thanks for adding the accessibility spec. 
You need to `require "axe-rspec"` in the rails_helper.rb so that the spec can use it and see the methods in the accessibility.spec.